syncano/models/backups.py

```diff
@@ -45,5 +47,25 @@ class Meta:
             'list': {
                 'methods': ['post', 'get'],
                 'path': '/backups/full/',
+            },
+            'restore': {
+                'methods': ['post'],
+                'path': '/restores/',
+            }
+        }
+
+    def restore(self):
+        properties = self.get_endpoint_data()
+        endpoint = self._meta.resolve_endpoint('restore', properties)
+        kwargs = {
+            'data': {
+                'backup': self.id
             }
         }
+        connection = self._get_connection()
+        response = connection.request('POST', endpoint, **kwargs)
+        restore_endpoint = response['links']['self']
+        restore_response = connection.request('GET', restore_endpoint)
+        while restore_response['status'] == 'running':
```

Comment:
This is also problematic - when you rework this - just return the Restore object and leave reload (for the status update) to the user - or provide method is_success - which will ask for the status by API.